lib/docker/container.go

```diff
@@ -35,7 +37,15 @@ func CreateContainer(containerCfg *types.ApplicationContainer) (string, error) {
 		Volumes: map[string]struct{}{
 			volume: {},
 		},
+		Healthcheck : &container.HealthConfig{
+			Test: []string{"CMD-SHELL", fmt.Sprintf("curl --fail http://localhost:%d/ || exit 1", containerCfg.ContainerPort)},
+			Interval: configs.ServiceConfig.Mizu.MetricsInterval*time.Second,
+			Timeout: 10*time.Second,
+			Retries: 3,
```

Comment:
Add a start period of 2 minutes to the configuration

Reply:
there's no such member in the HealthConfig struct

Reply:
There is, check this out https://godoc.org/github.com/docker/docker/api/types/container#HealthConfig